FutureNorthants/northants-design-system
# Update npm dependencies

Update all the npm dependencies, except React as it is a major version (to 18) and we need to read up on the changes for next.js and the frontend. 

## To Test
- Checkout this branch and `npm install`
- Run `npm outdated` and only React and react-dom should display in the list
- Run `npm run test` to run the test suite
- Run `npm run dev` to launch storybook and manually test components

## Testing in the Frontend
- Run `yalc publish` then go to the frontend and checkout `preprod_frontend` branch
- Run `yalc add northants-design-system && yarn install` to pull in the latest version into the frontend
- Run `yarn e2e:headless` to run the integration tests
- Run `npx cypress run-ct` to run component tests
- Run `yarn build && yarn start` then test frontend pages manually in the browser
